dotnet/GraphQL/Query.cs

```diff
@@ -152,6 +152,62 @@ public Query(IProductReviewService productReviewService)
                 }
             );
 
+            FieldAsync<SearchResponseType>(
+                "reviewByreviewDateTime",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "reviewDateTime", Description = "Review DateTime" },
+                    new QueryArgument<StringGraphType> { Name = "searchTerm", Description = "Search term" },
+                    new QueryArgument<IntGraphType> { Name = "from", Description = "From" },
+                    new QueryArgument<IntGraphType> { Name = "to", Description = "To" },
+                    new QueryArgument<StringGraphType> { Name = "orderBy", Description = "Order by" },
+                    new QueryArgument<StringGraphType> { Name = "status", Description = "Status" }
+                ),
+                resolve: async context =>
+                {
+                    string reviewDateTime = context.GetArgument<string>("reviewDateTime");
+                    string searchTerm = context.GetArgument<string>("searchTerm");
```

Comment:
The only parameter being used is reviewDateTime, the others should be removed or added to the search query.